.env.example

```diff
@@ -1,13 +1,7 @@
 MAINNET_DEPLOYER_PRIVATE_KEY="0x0000000000000000000000000000000000000000000000000000000000000000"
-ROPSTEN_DEPLOYER_PRIVATE_KEY="0x0000000000000000000000000000000000000000000000000000000000000000"
-RINKEBY_DEPLOYER_PRIVATE_KEY="0x0000000000000000000000000000000000000000000000000000000000000000"
-KOVAN_DEPLOYER_PRIVATE_KEY="0x0000000000000000000000000000000000000000000000000000000000000000"
 GOERLI_DEPLOYER_PRIVATE_KEY="0x0000000000000000000000000000000000000000000000000000000000000000"
 
 MAINNET_PROVIDER=https://eth-mainnet.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
-ROPSTEN_PROVIDER=https://eth-ropsten.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
-RINKEBY_PROVIDER=https://eth-rinkeby.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
-KOVAN_PROVIDER=https://eth-kovan.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 GOERLI_PROVIDER=https://eth-goerli.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 
```

Comment:
Add sepolia in here?

Reply:
Sure I could, but I was planning on adding Sepolia stuff in the future PR where we re-deploy to these contracts